### PR TITLE
Prerelease/2.4.0 stabilization toggle features panel

### PR DESCRIPTION
**Pull request by cre8ivepark**
## Overview
Bugfix for #7872 Toggle Features Panel is missing toggle state.

Updated the buttons to Toggle Buttons in these two prefabs:
- ToggleFeaturesPanel.prefab
- ToggleFeaturesPanelExamplesHub.prefab

Added a red color material for the Input Recording toggle state.

## Changes
- Fixes: #7872 

2020-05-14 12_09_09-Unity 2018 4 21f1 Personal -  PREVIEW PACKAGES IN USE  - HandInteractionExamples

### CODE REVIEW

**Comment by cre8ivepark** [comment]
I found that Input Recording feature is not working in MRTK Examples Hub scene. Verified that it is not the button/toggle features panel problem since it works in other scene. It could be related to the custom profile for the MRTK Examples Hub scene. I see there is Input Recording Service in the MRTK profile. Not sure why it shows Null Reference Exception.

NullReferenceException: Object reference not set to an instance of an object
Microsoft.MixedReality.Toolkit.Input.**InputRecordingService.PruneBuffer** () (at Assets/MRTK/Services/InputAnimation/InputRecordingService.cs:322)
Microsoft.MixedReality.Toolkit.Input.InputRecordingService.set_UseBufferTimeLimit (System.Boolean value) (at Assets/MRTK/Services/InputAnimation/InputRecordingService.cs:103)
Microsoft.MixedReality.Toolkit.UI.InputRecordingControls.ToggleRecording () (at Assets/MRTK/SDK/Features/UX/Scripts/Utilities/InputRecordingControls.cs:60)
UnityEngine.Events.InvokableCall.Invoke () (at C:/buildslave/unity/build/Runtime/Export/UnityEvent.cs:166)
UnityEngine.Events.UnityEvent.Invoke () (at C:/buildslave/unity/build/Runtime/Export/UnityEvent_0.cs:58)
Microsoft.MixedReality.Toolkit.UI.Interactable.SendOnClick (Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointer pointer) (at Assets/MRTK/SDK/Features/UX/Interactable/Scripts/Interactable.cs:1251)
Microsoft.MixedReality.Toolkit.UI.Interactable.TriggerOnClick () (at Assets/MRTK/SDK/Features/UX/Interactable/Scripts/Interactable.cs:1241)
Microsoft.MixedReality.Toolkit.PhysicalPressEventRouter.OnHandPressTriggered () (at Assets/MRTK/SDK/Features/UX/Scripts/PressableButtons/PhysicalPressEventRouter.cs:90)
UnityEngine.Events.InvokableCall.Invoke () (at C:/buildslave/unity/build/Runtime/Export/UnityEvent.cs:166)
UnityEngine.Events.UnityEvent.Invoke () (at C:/buildslave/unity/build/Runtime/Export/UnityEvent_0.cs:58)
Microsoft.MixedReality.Toolkit.UI.PressableButton.UpdatePressedState (System.Single pushDistance) (at Assets/MRTK/SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs:480)
Microsoft.MixedReality.Toolkit.UI.PressableButton.UpdateTouch () (at Assets/MRTK/SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs:304)
Microsoft.MixedReality.Toolkit.UI.PressableButton.Update () (at Assets/MRTK/SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs:289)

**Comment by cre8ivepark** [comment]
## Verified that Input Recording issue on Examples Hub is also in 2.3.0 release. I'll open a separate bug.

**Comment by keveleigh** [comment]
@cre8ivepark While you're editing this panel, could you remove / hide the Gaze Source button please? It was requested and I filed #7884 to track it.

I can also update it myself after this is merged if you don't have time!

**Comment by cre8ivepark** [comment]
@keveleigh Oh, I see. Let me remove it here.

**Comment by cre8ivepark** [comment]
- Removed Gaze Source button. 
- Updated layout back to single row since we have five buttons.
- Updated ToggleFeaturesPanelExamplesHub as prefab variant. 
- Updated Examples Hub to use this prefab. 
- Updated material color to red for the Recording State backplate material.

### ToggleFeaturesPanel.prefab
2020-05-15 15_16_30-Unity 2018 4 21f1 Personal -  PREVIEW PACKAGES IN USE  - HandInteractionExamples

### ToggleFeaturesPanelExamplesHub.prefab
2020-05-15 15_16_01-Unity 2018 4 21f1 Personal -  PREVIEW PACKAGES IN USE  - MRTKExamplesHub unity -

File: Assets/MRTK/Examples/Experimental/ExamplesHub/Prefabs/ToggleFeaturesPanelExamplesHub.prefab.meta [hunk 1]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 26b0c24187a40734a89bfc6c224c8c60
+guid: ab9984c5afa3beb4b882dac9f6eed4ac
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Experimental/ExamplesHub/Prefabs/ToggleFeaturesPanelExamplesHub.prefab.meta:2]: Was this updated intentionally or should we try to keep it with the old GUID?

**Comment by cre8ivepark** [comment]
I am doing another update to the ToggleFeaturesPanelExamples Hub to resolve #7890

**Comment by cre8ivepark** [comment]
Opened a new PR #7902 to address #7890 issue as well. Also reverted GUID change in the new PR.

**Pull request closed without merging**